Lib/test/test_httpservers.py

```diff
@@ -489,6 +489,9 @@ def test_get(self):
         self.check_status_and_reason(response, HTTPStatus.NOT_FOUND)
         response = self.request('/' + 'ThisDoesNotExist' + '/')
         self.check_status_and_reason(response, HTTPStatus.NOT_FOUND)
+        os.makedirs(os.path.join(self.tempdir, 'spam', 'index.html'))
+        response = self.request(self.base_url + '/spam/')
+        self.check_status_and_reason(response, HTTPStatus.OK)
```

Comment:
Could you add a test with a directory named `index.html`?  It would fail without the change and pass with it.

Reply:
That test is creating a directory for index.html, as os.makedirs only creates directories.
Should I make it clearer by putting a file within it?

I did it under a subdirectory (spam) as it makes it clearer that the issue isn't related to it being the / directory (though that is also affected).

Reply:
Ah then you’re fine!